### PR TITLE
feat(aps): add macos beta upgrade doc

### PR DESCRIPTION
**Pull request by bene2k1**
### Your checklist for this pull request

- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

**Review by RoRoJ — APPROVED** [review]

File: bare-metal/apple-silicon/how-to/upgrade-macos-to-beta.mdx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  h1: How to upgrate to a beta version of macOS on a Mac mini
+  paragraph: This page explains how to upgrade the macOS of a Scaleway Mac mini to a beta version.
+tags: apple-silicon mac-mini beta macos
+dates:
+  validation: 2024-07-30
+  posted: 2021-07-30
+categories:
+  - bare-metal
+---
+
```
Comment (jcirinosclwy) [inline bare-metal/apple-silicon/how-to/upgrade-macos-to-beta.mdx:15]: This How-to doesn't have any requirements?

File: bare-metal/apple-silicon/how-to/upgrade-macos-to-beta.mdx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+2. Back up your Mac mini.
+    <Message type="important">
+        Ensure you [back up your data and files](/faq/apple-silicon/#how-can-i-back-up-my-mac-mini) before installing beta versions of macOS.
+    </Message>
+
+3. Sign up for Apple's **Beta Software Program**:
+   - Visit [beta.apple.com](https://beta.apple.com), click **Sign Up** and log in with your Apple ID.
+        <Message type="note">
+            An Apple ID is required to join the beta software program. Create a new one if you do not have one yet.
```
Comment (jcirinosclwy) [inline bare-metal/apple-silicon/how-to/upgrade-macos-to-beta.mdx:28]:
```suggestion
            An Apple ID is required to join the beta software program. Create an ID if you do not have one yet.
```

File: bare-metal/apple-silicon/how-to/upgrade-macos-to-beta.mdx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        </Message>
+
+4. Click **Enroll your Mac** in the **Get Started** section of Apples website to register your Mac for the beta software program.
```
Comment (jcirinosclwy) [inline bare-metal/apple-silicon/how-to/upgrade-macos-to-beta.mdx:31]:
```suggestion
4. Click **Enroll your Mac** in the **Get Started** section of Apple's website to register your Mac for the beta software program.
```

Comment (jcirinosclwy) [reply bare-metal/apple-silicon/how-to/upgrade-macos-to-beta.mdx:31]: updating the casing here cause above it was in uppercase

**Review by jcirinosclwy — APPROVED** [review]

**Review by nerda-codes — APPROVED** [review]

**Pull request merged**